packages/dart_frog/test/src/serve_test.dart

```diff
@@ -88,5 +89,81 @@ void main() {
+cC/DsUVxKGkKcSQPXR3GGNp/VWbs7w+qqTD/PHI5J4r69GBR5ZXu8l24i8UCAwEA
+AaNeMFwwDgYDVR0PAQH/BAQDAgWgMBMGA1UdJQQMMAoGCCsGAQUFBwMBMB8GA1Ud
+IwQYMBaAFDptHqlxKExeYwjJh6dg0wXdlaocMBQGA1UdEQQNMAuCCWxvY2FsaG9z
+dDANBgkqhkiG9w0BAQsFAAOCAYEANRtOVP1GypaeTXBeKnoOGycPexnVIwpPgysM
+v2LCsoQzWbZJkOeZadjOODBoJSCblLAQvx6deo8isGOYJO73lrLuX3CldVBuwP1q
+nBlotfJN+kk42ixT+ETxD/vSOY6CxClo4pw4f8SXlyZ6RX3pKTyXLC5vud4z4kIt
+X5SUy6moZfagBn2lntBx1qnKpXYPsaQQN23dDJE44PQX9o78ZffUn81E39sGJFQt
+V5ktO6WJh+5J+6TdEd1xnhC51QSJhRNuD2H8H5PVhoAWiyMK8te+lMUpK6x4Mlbo
+eOjaAAhQ8UcVmvJrtt5GoDh9Bl/jB60gVGL0XFr9E1b3DZWgMpgyzBreGlpXPL87
+47P6RMmzpAp/efrY+scKk9UH/6CElsWt2lYrv4XUBjsCXdrGrYpuG3aS5ohowvWd
+lM5/EJeo2XQrb5AOE+mGMOZrI1F113IjZkP73OVrUyAKg5vsHDvHwRQ/blz3W2fG
+iTz7oBJ+5dtzLcZNFkDgNhBGKSu8
+-----END CERTIFICATE-----''';
+      const key = '''
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQC4PHE6IrELM/JS
+rIo2uAt1ngmtjFdtGZOkhbNZEFPv2f/XchMUJQ05zDfHfA75Io/tqw2uz6IgBUZF
+mn5tAQrSxL+f/leLAgKW4W6j+Iqp7QO1QwuhogwIy/8qoGfW7Wh571Kig12DMrsl
+S9HVjiCxYT6g3/9K95sElV13BbEL2ON5Wdf7bXgojjTJlojy/cLiCWtMKgOi8LRs
+4y08l1Jju7WCwE9TM9V46mZ4UluNRUyyowNH7CgDKdbc2mZfA8zJwJqyzhHtxPiV
+c04cuD3w9yfPmnAvw7FFcShpCnEkD10dxhjaf1Vm7O8Pqqkw/zxyOSeK+vRgUeWV
+7vJduIvFAgMBAAECggEAfa3lw7XUtoK6RMGlC4zjbFnh2j0JishO2oXGgfRMfitl
+hvAvqadY7VutlWzAvh1gt83faKgFvfg7JtIsemmim4NSAW+9AnvdjlW8ZyjuVtrz
+k5xn+9wSf5HBwK9qBskvYzbqVShuC0j5N4kQXLE3BioDUjVb7yUX37mQ59e/Hgev
+TlmBRlQ1m6lXTu7slsOhrCF830acFullZcFQhhPeh9d4f32vMiCyCd3BhC0r50Jd
+45qdsQO1IGXKT4HYhdwI66BzV03kBM6rlsOltuhw0e+4ygXbLB5NOiuHNXz8Vju3
+d7Q13rc79udiXLcLxN+RgdVRR0F+imgRu/zq/gRGBQKBgQDUdfkFvxTYwmLV4OUD
+c8I/hjRX+RDgRfze0JhByp5VfGZ4NRhfo3JORmUU2fI+3jKxFineabtIl31ebBCc
+yK7hZPsdgluiLjrt/bAxOq4v47hOpyMQI9qxiBrJYDpID+emT/K3jfA0gYmZfl/c
+vYR5jSSD3d29UW3zqHNxni+d1wKBgQDd/cGVkWCH5k3k3Y94fSn//vbfECx0xltp
+025+rJ+go7T0ywSfNM3g5n1lt8jE0yJziiWF5PIM0qRo4YBWcPoI8sdFIaS+rHv7
+Oh1vSnX7vL5t0RNTjVpxpcxqAut2v03tMcuJo+8llEq+midp8beAmzsufKBONauX
+zCEk30MXwwKBgQC9qF5XEd9DLCtcb7kgHsrtOBkr2wuEmRWFtcHlIUG8YCN89TC/
+10EnvNFpDrGgC2xHBtjzUYE86PaiPmeJ/d+XFzTPf9na6dfzMX6CQ7bQy0Bw/eRf
++RG1XyFCWKNORtxsa3vo/UzLIkO6AMUEYS2L8EIDcSALa1ByrRH4/9PT2wKBgQDZ
+f+2ysIxmuoQpL9eJEwEam+GfbgZQp6QbDJgfLtz7lEoQ6fTuU9s/djT4e1gPWFpR
+39Gh3U42uA9z3zVR/EFOkSgimLMESpTy8d6zEr6EVkox6H5KB53M6chdOd0gLJGa
+S4aDpgYCyMdu9jSVvcmwDOewRVT/K+CiytLSgJkI5wKBgE9Lun9U4sW8uKDL0q9g
+GhAvwnx67SU85gBKK+C1P9yBp8QGQEjFHtRBLUh5R4lXBvPW9e+VuS0vcj6QKGuy
+4fWk4JDF7cwbR/HqlyNQrynnSIm1qats3Oe2AiJzKNXMzQj/IiVgF/cp5z1qjkKZ
+SFTrELxay/xfdivEUxK9wEIG
+-----END PRIVATE KEY-----''';
+
+      final securityContext = SecurityContext()
+        ..useCertificateChainBytes(utf8.encode(chain))
+        ..usePrivateKeyBytes(utf8.encode(key));
+      final server = await serve(
+        (_) => Response(),
+        'localhost',
+        3000,
+        securityContext: securityContext,
+      );
+      final client = HttpClient()
+        ..badCertificateCallback =
+            (X509Certificate cert, String host, int port) => true;
```

Comment:
```suggestion
            (X509Certificate cert, String host, int port) => cert.pem == chain;
```

You also need to add a new line to chain.